docs/learning/ui/framework/Frontstages.md

```diff
@@ -8,153 +8,136 @@ A **Frontstage** is a full-screen configuration designed to enable the user to a
 |**Nested** | is accessed from a primary frontstage. It may use all zones and panels, but instead of the App button, the Tool Widget contains a Back button to return to the primary frontstage.
 |**Modal** | is accessed from another frontstage or the Backstage. It may contain any content along with a Back button. It does not use zones or stage panels. It is useful for application settings and data management user interfaces.
 
-## Frontstages in UI 2.0/"App UI"
+## Frontstages in App UI
 
 With the release of the `iTwin.js 2.0`, new UI components are available that provide a new look and feel for iTwin Apps. The new look and feel was initially referred to as `UI 2.0` and now has the more formal name of a `App UI`. The two primary goals of `App UI` are to limit the amount of UI components that obscure the iModel content and to ensure that Extensions can augment the UI provided by the host IModelApp.
```

Comment:
Can we update this section to not refer directly to Extensions but rather say that augmentation of the UI can be lazily-loaded into the application? Or something similar...